source/jsonnet/england-wales/household/blocks/visitor/visitor_usual_address_country.jsonnet

```diff
@@ -21,7 +21,7 @@ local rules = import 'rules.libsonnet';
         label: 'Current name of country',
         description: 'Enter your own answer or select from suggestions',
         mandatory: false,
-        suggestions_url: 'https://cdn.eq.census-gcp.onsdigital.uk/data/v1.0.0/countries-of-birth.json',
+        suggestions_url: 'https://cdn.eq.census-gcp.onsdigital.uk/data/v4.0.0/gb/{language_code}/countries-of-birth.json',
```

Comment:
I think it makes sense to specify the `version` at build time, so we don't have to replace `v4.0.0` in all places.

Reply:
I added VERSION const to `resolve_suggestions_urls` and `version` placeholder to schema's suggestions, is that what you requested?👍